tsconfig.json

```diff
@@ -0,0 +1,7 @@
+{
+  "compilerOptions": {
+    "module": "commonjs",
+    "strict": true,
+    "target": "es6"
+  }
+}
```

Comment:
I copied this from elsewhere but not sure that I need even this much, since I'm only running the "assertion tests".